### DISCUSSION TITLE
How I can host my own pocketbase with SSL on my Server?

### DISCUSSION THREAD

**Discussion by miguelgargallo** [pocketbase/pocketbase#2076] [Q&A]
How I can host my own pocketbase with SSL on my Server? the problem is that I don't know for sure where to put the bp file to serve the server with SSL on Arch

**Comment by ganigeorgiev** [comment]
The executable can be placed technically anywhere you want. You can follow the 4-step guide from https://pocketbase.io/docs/going-to-production/#minimal-setup.


When PocketBase is started with the `--https="yourdomain.com:443"` flag, it will automatically issue a Let's encrypt certificate for your domain.

If you want to use a custom certificate, at the moment this is not possible and you'll have to put PocketBase behind a reverse proxy (nginx, apache, caddy, etc.) and manage the certificate via certbot or some other tool. You can find an example nginx config in https://pocketbase.io/docs/going-to-production/#using-reverse-proxy.
